### PR TITLE
Add RTL language authoring option

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@milton could you please quickly review how I've done this and then I'll do the same for Pages.

File: project/templates/admin/cms/category/change_form.html [hunk 1]
```diff
@@ -46,6 +54,27 @@
...
+                $(document).ready(function() {
+                    var LTR_radio = document.getElementById("lang_LTR");
+                    LTR_radio.checked = true; //default is LTR
+                    LTR_radio.onclick = function() {
+                        $("#id_title").attr("dir", "ltr");
+                        $("#id_slug").attr("dir", "ltr");
+                        $("#id_subtitle").attr("dir", "ltr");
+                    }
+                    var RTL_radio = document.getElementById("lang_RTL");
+                    RTL_radio.onclick = function() {
```
Comment (miltontony) [inline project/templates/admin/cms/category/change_form.html:68]: @nathanbegbie you could also use `$("#lang_RTL").click(function(){})`

Comment (miltontony) [inline project/templates/admin/cms/category/change_form.html:68]: just a pro tip, no need to change. It's exactly the same thing under the hood

**Comment by nathanbegbie** [comment]
@milton @smn @steve All the textboxes are done with RTL support. So, for someone creating content in RTL, text entry is supported. The layout of the site will still appear Left to Right though. How user friendly are we planning on making cms-django (mirroring, whether the translated form is postition left or right etc), given its impending destruction?

**Comment by SteveBarnett** [comment]
@nathanbegbie I think RTLing the whole of cms-django is a big job, and that we don't want to spend that amount of time on this (soon to be defunct) CMS. On the other hand, I suspect someone may have done that work already and rolled it out as a Django add-on.

**Comment by nathanbegbie** [comment]
@miltontony @smn: +1 ?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**